nomad/start_nomad.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Think of those poor NixOS users that have bash somewhere else on the path (mostly me)